test/integration/integration_config.py

```diff
@@ -0,0 +1,24 @@
+import os
+import yaml
+
+
+class IntegrationConfig:
+    def __init__(self):
+        dir_path = os.path.dirname(os.path.realpath(__file__))
+        path = os.path.join(dir_path, "clarity_steps.yml")
```

Comment:
Ah. I missed that you did a `config = IntegrationConfig()` in ` test/integration/__init__.py`. :+1: